packages/sn-filter-pane/src/components/FoldedListbox/DimensionIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tooltip } from '@mui/material';
+import React from 'react';
+import { stardust } from '@nebula.js/stardust';
+import DrillDown from './drillDown';
+import Cyclic from './cyclic';
+
+interface Props {
+    isRtl: boolean;
+    translator?: stardust.Translator;
+    type: string;
+}
+
+const DimensionIcon = ({
+  type, translator, isRtl,
+}: Props) => {
+  const delay = 2000;
+  const style = { fontSize: '12px', padding: isRtl ? '0 0 0 6px' : '0 6px 0 0' };
+  switch (type) {
+    case 'H': return (
+        <Tooltip title={translator?.get('Listbox.DrillDown')} enterDelay={delay}>
+          <div>
+            <DrillDown style={style} />
+          </div>
+        </Tooltip>
+    );
+    case 'C': return (
+      <Tooltip title={translator?.get('Listbox.Cyclic')} enterDelay={delay}>
```

Comment:
We don't have any locale/translations in this repo at the moment, so I guess the proper way is to introduce that. Maybe look at how it's done in qlik-trial/nebula-tables -> packages/sn-table/src/locale and do in a similar way here...

Reply:
But there is a "Listbox.Drilldown" above this one which is translated. Do you know where that from? because this is not the first translation string introduced here.

Reply:
That's right... It will only have access to that translation when running inside sense-client though. We should actually have introduced the translations in here when we added that one to make sure it gets translated when rendering the chart outside of the sense-client as well.

Reply:
Ok, I see I will checkout the nebula-tables. Thanks

Reply:
@veinfors I am thinking if I could do the translation thing in another PR. I want to get this change into the windows release.

Reply:
I have introduced the translation and do I need to do something so that translation people get notified for updating the translation to other languages ?

Reply:
Sent you a link on slack about this.